pom.xml

```diff
@@ -177,7 +177,8 @@
 		<log4j2.version>2.24.2</log4j2.version>
 		<jmh.version>1.37</jmh.version>
 		<classgraph.version>4.8.179</classgraph.version>
-		
+		<logcaptor.version>2.10.0</logcaptor.version>
```

Comment:
please move this to line 178, just below `log4j2.version`, so that it better fits the "group" of dependencies.